src/correct-round.rkt

```diff
@@ -26,9 +29,16 @@
       (timeline-start!/unsafe
        'mixsample "backward-pass" (* (*sampling-iteration*) 1000)))
     (define first-iter? (zero? (*sampling-iteration*)))
-    (if first-iter?
-        (vector-fill! vrepeats #f)
-        (backward-pass ivec varc vregs vprecs vstart-precs rootvec rootlen vrepeats))
+    (match first-iter?
+      [#t (when (and (> iter-count 0) (*use-sharing-exponents*))
+            ; Get converged precisions with slack = 0
+            (parameterize ([*sampling-iteration* -1])
+              (backward-pass ivec varc vregs vprecs vbase-precs rootvec rootlen vrepeats))
+            (update-vstart-precs vstart-precs vprecs))
```

Comment:
I see, this saves the final precisions on the _first_ iter of the _next_ point. Clever.

Reply:
Thanks, it also saves precisions only if the previous point was problematic. This way it doesn't affect the first pass when the benchmark doesn't have problematic points